test/decorator-observable.spec.js

```diff
@@ -108,15 +112,41 @@ describe('observable decorator', () => {
     instance.value = newValue;
     expect(instance.customHandler).toHaveBeenCalledWith(newValue, oldValue, 'value');
   });
-  
+
   it('should create an enumerable accessor', () => {
     const instance = new class {
       @observable value;
     };
-    const keys = [];
-    for (let p in instance) {
-      keys.push(p);
-    }
-    expect(keys).toContain('value');
+
+    expect(Object.getPrototypeOf(instance).propertyIsEnumerable('value')).toBe(true);
+  });
+
+  describe('inner property', () => {
+    it('should exist', () => {
+      const instance = new class {
```

Comment:
Why is the const called `instance` when controversially it seems to be a class, not instance of the class?